internal/api/internal/client_test.go

```diff
@@ -0,0 +1,39 @@
+package internal
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/stretchr/testify/require"
+)
+
+func TestClient_Bootstrap(t *testing.T) {
+	s := NewServer("", nil, hclog.NewNullLogger())
+
+	testServer := httptest.NewServer(s)
+	defer testServer.Close()
+
+	tests := []struct {
+		name           string
+		wantStatusCode int
+	}{
+		{
+			name:           "stub",
+			wantStatusCode: http.StatusNotImplemented,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			client, err := CreateClient(ClientConfig{
+				Server: testServer.URL,
+				Base:   "/",
+			})
+			require.NoError(t, err)
+			_, err = client.Bootstrap(context.Background())
+			require.Error(t, err)
```

Comment:
it's part of the generated code -- `Bootstrap` is a remote call to the internal endpoint `/api/internal/bootstrap`.

Reply:
specifically it's a call the higher level wrapper [here](https://github.com/hashicorp/consul-api-gateway/pull/346/files#diff-dba36cfc1cf05c771d54926ec4fe7f1a1a9378aa18518599f31dde6205ac80b6R44), that is making the stock generated [`BootstrapWithResponse`](https://github.com/hashicorp/consul-api-gateway/pull/346/files#diff-8390553286e923f2997a73afe5b3ce6632483f98c06d4391bf68d68aaa20512fR223) a little easier to use by separating out JSON-based error messages (since they implement the `error` interface) from successful API calls.